src/ConfigServiceBase.ts

```diff
@@ -34,6 +33,14 @@ export class RefreshResult {
   }
 }
 
+/** Specifies the possible states of the local cache. */
+export enum ClientCacheState {
+  NoFlagData,
+  HasLocalOverrideFlagDataOnly,
+  HasCachedFlagDataOnly,
```

Comment:
Nem tudom, hogy érdemes lenne itt a névben jobban jelezni, hogy ez egy expired dolog. Elsőre nekem úgy jön le, hogy ez egy jó állapot. Szerinted?

Reply:
Igen, az expired szerintem is sokkal kifejezőbb lenne. Csak annyi a kérdés, hogy meg tudjuk-e ezt tenni breaking change nélkül. Talán igen, kiderítem, hogy mit tehetünk az ügyben.

Reply:
Ezt a megoldást tudtam kitalálni, amivel (többé-kevésbé) tudjuk tartani a backward compatibility-t: https://github.com/configcat/common-js/pull/92/commits/14ebf9f9232ee7ee28522c8cc3b5d16bb2254385#diff-60c48971c3c3e6b9062ebd1db0079dda589d89c32988c307e5954bef5efcb8ebR40-R42

Illetve az "expired" helyett a "stale" elnevezést javaslom, mert Manual Poll esetén talán az egy fokkal jobb kifejezés (mivel ott az expired fogalom by design nem létezik).

@z4kn4fein Részedről is rendben van ez a módosítás?

Reply:
A stale helyett inkább valami mással mennék, expired, vagy nem tudom, a stale kifejezés már "foglalt" a zombie/stale flag által :(

Reply:
Tudod mit, jól van az úgy, ahogy volt, i can live with that, és akkor nem kell semmi obsolete téma. Bocsika.